test/integrations/destinations/intercom/dataDelivery/business.ts

```diff
@@ -0,0 +1,147 @@
+import {
+  generateMetadata,
+  generateProxyV0Payload,
+  generateProxyV1Payload,
+} from '../../../testUtils';
+
+const commonHeaders = {
+  'Content-Type': 'application/json',
+  Authorization: 'Bearer intercomApiKey',
+  Accept: 'application/json',
+  'Intercom-Version': '1.4',
+};
+
+const createUserPayload = {
+  email: 'test_1@test.com',
+  phone: '9876543210',
+  name: 'Test Name',
+  signed_up_at: 1601493060,
+  last_seen_user_agent: 'unknown',
+  update_last_request_at: true,
+  user_id: 'test_user_id_1',
+  custom_attributes: {
+    anonymousId: '58b21c2d-f8d5-4410-a2d0-b268a26b7e33',
+    key1: 'value1',
+    'address.city': 'Kolkata',
+    'address.state': 'West Bengal',
+    'originalArray[0].nested_field': 'nested value',
+    'originalArray[0].tags[0]': 'tag_1',
+    'originalArray[0].tags[1]': 'tag_2',
+    'originalArray[0].tags[2]': 'tag_3',
+    'originalArray[1].nested_field': 'nested value',
+    'originalArray[1].tags[0]': 'tag_1',
+    'originalArray[2].nested_field': 'nested value',
+  },
+};
+
+const commonRequestParameters = {
+  JSON: createUserPayload,
+  headers: commonHeaders,
+};
+
+const expectedStatTags = {
+  destType: 'INTERCOM',
+  destinationId: 'default-destinationId',
+  errorCategory: 'network',
+  errorType: 'retryable',
+  feature: 'dataDelivery',
+  implementation: 'native',
+  module: 'destination',
+  workspaceId: 'default-workspaceId',
+};
+
+const metadataArray = [generateMetadata(1)];
+
+export const testScenariosForV0API = [
+  {
+    id: 'intercom_v0_other_scenario_1',
+    name: 'intercom',
+    description: '[Proxy v0 API] :: Scenario for testing Request Timeout error from destination',
```

Comment:
These scenarios should be add as `other.ts` file separtely

Reply:
addressed